bedrock/mozorg/templates/mozorg/about/governance/organizations.html

```diff
@@ -14,17 +14,23 @@ <h1 class="title-shadow-box">{{_('Mozilla Organizations')}} </h1>
 
 <p>{{_('There are several organizations that support the Mozilla community and ')}}<a href="{{ url('mozorg.about.manifesto') }}">{{_('Mozilla’s principles.</a> They include the non-profit Mozilla Foundation as well as the wholly owned taxable subsidiary, the Mozilla Corporation. Mozilla considers itself a <a href="http://commonspace.wordpress.com/category/hybrid/">hybrid organization</a>, combining non-profit and market strategies to ensure the Internet remains a shared public resource.')}}</p>
 
-<p>{{_('This page provides information about the various organizations and their relationships to each other. If you have any questions about this, please feel free to <a href="//www.mozilla.org/contact">contact us</a>')}}</p>
+<p>{% trans contact_url=url('mozorg.contact.contact-landing') %}
+This page provides information about the various organizations and their relationships to each other. If you have any questions about this, please feel free to <a href="{{ contact_url }}">contact us</a>
+{% endtrans %}</p>
 
 <h2>{{_('Mozilla Foundation')}}</h2>
 
-<p>{{_('The <a href="//www.mozilla.org/foundation/">Mozilla Foundation</a> is a California non-profit corporation exempt from Federal income taxation under IRC 501(c)(3). The Foundation supports the existing Mozilla community and oversees Mozilla’s governance structure. It also actively seeks out new ways for people around the world to recognize and steward the Internet as a critical public resource.')}}</p>
+<p>{% trans foundation_url=url('foundation.index') %}
+The <a href="{{ foundation_url }}">Mozilla Foundation</a> is a California non-profit corporation exempt from Federal income taxation under IRC 501(c)(3). The Foundation supports the existing Mozilla community and oversees Mozilla’s governance structure. It also actively seeks out new ways for people around the world to recognize and steward the Internet as a critical public resource.
+{% endtrans %}</p>
 
-<p>{{_('<a href="//www.mozilla-japan.org/">Mozilla Japan</a> is a separate non-profit organization that promotes Mozillas products and mission in Japan and is affiliated with the Mozilla Foundation.')}}</p>
+<p>{{_('<a href="http://www.mozilla-japan.org/">Mozilla Japan</a> is a separate non-profit organization that promotes Mozillas products and mission in Japan and is affiliated with the Mozilla Foundation.')}}</p>
```

Comment:
mozilla-japan.org redirects to mozilla.jp, should we go ahead and change it here? (and mozilla.jp isn't https either)